express/parsers/__init__.py

```diff
@@ -10,6 +12,11 @@ def __init__(self, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
 
+    def unix_path_names(self, full_path: str) -> Tuple[str, str]:
+        """Return unix-like dirname and basename from a full path"""
```

Comment:
To be consistent with the rest of the codebase, let's keep newlines in the docstring, pls